### DISCUSSION TITLE
Not compatible with neovide. Throw error:"copilot copilot_node_comman(node) is not executable"

### DISCUSSION THREAD

**Discussion by GinmusWJQ** [zbirenbaum/copilot.lua#158] [Q&A] [answered]
- My `neovim` verison: `0.10.0-dev`
- My `neovide` version: `0.10.4`
- My Envirement: WSL2`Linux version 5.10.16.3-microsoft-standard-WSL2 (oe-user@oe-host) (x86_64-msft-linux-gcc (GCC) 9.3.0, GNU ld (GNU Binutils) 2.34.0.20200220) #1 SMP Fri Apr 2 22:23:49 UTC 2021`

Here is my copilot.lua configuration
```lua
{
  "zbirenbaum/copilot.lua",
  cmd = "Copilot",
  build = ":Copilot auth",
  opts = {
    suggestion = { enabled = false },
    panel = { enabled = false },
  },
}
```
I excute `neovim --wsl` to use my neovim configuration that is in wsl2.
But just throw error message "copilot copilot_node_comman(node) is not executable"

**Comment by MunifTanjim** [comment, marked as answer]
Neovide doesn't have access to `node` executable. That's why it's showing the error.

Is Neovide installed inside WSL or outside? Make sure Node.js is installed in the same environment.

How are you launching Neovide?

Either make sure it has access to the `node` executable (via path environment variable).

Or specify full path to Node.js executable to `copilot_node_command` in setup:

```lua
{
  "zbirenbaum/copilot.lua",
  cmd = "Copilot",
  build = ":Copilot auth",
  opts = {
    suggestion = { enabled = false },
    panel = { enabled = false },

    copilot_node_command = "/path/to/node"
  },
}
```

@GinmusWJQ
